cantor-s3/src/main/java/com/salesforce/cantor/s3/CantorOnS3.java

```diff
@@ -15,15 +15,18 @@
 
 import java.io.IOException;
 
+/**
+ * This implementation is designed to only use a single s3 bucket.
+ */
 public class CantorOnS3 implements Cantor {
     private final Objects objects;
 
     public CantorOnS3(final AmazonS3 s3Client) throws IOException {
-        this(s3Client, "default");
+        this(s3Client, "warden-cantor");
```

Comment:
@AlexanderKouthoofd let's remove this constructor and always expect bucket name to be passed in